lib/web.coffee

```diff
@@ -1,23 +1,93 @@
-_ = require 'underscore-plus'
-url = require 'url'
-_str = require 'underscore.string'
+# atom clean up work
+{CompositeDisposable} = require 'atom'
```

Comment:
Removed the deprecated protocol/application programming interface entirely. Nothing is require(d) except 'atom'